README.md

```diff
@@ -1,231 +1,39 @@
-
-With command line
-
-`sudo /opt/nvidia/jetson-io/config-by-hardware.py -n 2="Jetson RealSense Camera D457"`
-
-Expected:
-```
-nvidia@ubuntu:~$ sudo /opt/nvidia/jetson-io/config-by-hardware.py -n 2="Jetson RealSense Camera D457"
-Configuration saved to /boot/tegra234-camera-d4xx-overlay.dtbo.
-Reboot system to reconfigure.
-```
-Enable d4xx autoload:
-
-`echo "d4xx" | sudo tee /etc/modules-load.d/d4xx.conf`
-
-`sudo depmod`
-
----
-
-</details>
-
-## Install kernel and D457 driver to Jetson AGX Xavier
-
-1. Install the kernel and modules
-
-1.1 If building with `build_all_deb.sh`
-
-Copy the Debian package `linux-image-5.10.104-d457_5.10.104-d457-1_arm64.deb` to the Jetson AGX Xavier board and install with `sudo dpkg -i linux-image-5.10.104-d457_5.10.104-d457-1_arm64.deb`. The header, libc-dev, dbg and firmware packages are optional.
-
-1.2 If building with `build_all.sh`
-
-The necessary files are:
-
-- kernel image `images/<JetPack_version>/arch/arm64/boot/Image`
-- dtb `images/<JetPack_version>/arch/arm64/boot/dts/nvidia/tegra194-p2888-0001-p2822-0000.dtb`, or `images/4.6.1/arch/arm64/boot/dts/tegra194-p2888-0001-p2822-0000.dtb` for JetPack 4.6.1.
-- D457 driver `images/<JetPack_version>/drivers/media/i2c/d4xx.ko`
-- UVC Video driver `images/<JetPack_version>/drivers/media/usb/uvc/uvcvideo.ko`
-- V4L2 Core Video driver `images/<JetPack_version>/drivers/media/v4l2-core/videobuf-core.ko`
-- V4L2 VMalloc Video driver `images/<JetPack_version>/drivers/media/v4l2-core/videobuf-vmalloc.ko`
-
-And copy them to the right places:
-```
-sudo cp Image /boot
-sudo cp tegra194-p2888-0001-p2822-0000.dtb /boot/dtb
-sudo cp d4xx.ko /lib/modules/5.10.104-tegra/kernel/drivers/media/i2c/
-sudo cp uvcvideo.ko /lib/modules/5.10.104-tegra/kernel/drivers/media/usb/uvc/
-sudo cp videobuf-core.ko /lib/modules/5.10.104-tegra/kernel/drivers/media/v4l2-core/
-sudo cp videobuf-vmalloc.ko /lib/modules/5.10.104-tegra/kernel/drivers/media/v4l2-core/
-sudo depmod
-```
+Please follow the [instruction](https://docs.nvidia.com/sdk-manager/install-with-sdkm-jetson/index.html) to flash JetPack to the Jetson AGX Xavier with NVIDIA SDK Manager or other methods NVIDIA provides. Make sure the board is ready to use.
 
-2. Edit `/boot/extlinux/extlinux.conf` primary boot option's LINUX/FDT lines to use built kernel image and dtb file:
+**NOTE** : On Jetsons with modified factory setup modifications to build and deploy steps should be made by developer.
 
-```
-LINUX /boot/Image-5.10.104-d457
-FDT /usr/lib/linux-image-5.10.104-d457/tegra194-p2888-0001-p2822-0000.dtb
-```
+# JetPack build
 
-3. Make D457 I2C module autoload at boot time: `echo "d4xx" | sudo tee /etc/modules-load.d/d4xx.conf`
+- [JetPack 6.0](./README_JP6.md#build-environment-prerequisites)
+- [JetPack 5.1.2](./README_JP5.md#build-kernel-dtb-and-d457-driver)
+- [JetPack 5.0.2](./README_JP5.md#build-kernel-dtb-and-d457-driver)
+- [JetPack 4.6.1](./README_JP4.md#build-kernel-dtb-and-d457-driver)
```

Comment:
Can we add the deserialize refresh command?
It was usefull
![image](https://github.com/IntelRealSense/realsense_mipi_platform_driver/assets/64067618/d7efc3ce-5326-4e9d-b13f-999b919867df)

Reply:
https://github.com/IntelRealSense/realsense_mipi_platform_driver/blob/b072985a90f98324c3fe41d400b7c7f60a3344f0/README.md?plain=1#L108

Reply:
This is relevant only for jp4